cypress/e2e/SuccessContent.cy.ts

```diff
@@ -0,0 +1,131 @@
+import { StatusCodes } from 'http-status-codes';
+
+import { API_ROUTES } from '@graasp/query-client';
+
+import { SIGN_IN_PATH, SIGN_UP_PATH } from '../../src/config/paths';
+import {
+  BACK_BUTTON_ID,
+  RESEND_EMAIL_BUTTON_ID,
+  SUCCESS_CONTENT_ID,
+} from '../../src/config/selectors';
+import { MEMBERS } from '../fixtures/members';
+
+describe('Success Content Validation', () => {
+  it('Sign In - Back Button', () => {
+    const { GRAASP, GRAASP_OTHER } = MEMBERS;
+    cy.visit(SIGN_IN_PATH);
+
+    cy.intercept(API_ROUTES.SIGN_IN_ROUTE, ({ reply }) => {
+      return reply({
+        statusCode: StatusCodes.NO_CONTENT,
+      });
+    });
+
+    cy.get(`#${SUCCESS_CONTENT_ID}`).should('not.exist');
+
+    // Siging in with a valid email
+    cy.signInAndCheck(GRAASP);
+
+    cy.get(`#${SUCCESS_CONTENT_ID}`).should('be.visible');
+
+    cy.get(`#${BACK_BUTTON_ID}`).click();
+
+    cy.get(`#${SUCCESS_CONTENT_ID}`).should('not.exist');
+
+    cy.url().should('include', SIGN_IN_PATH);
+
+    // check if it's possible to sign in and use back button again
+    cy.signInAndCheck(GRAASP_OTHER);
+
+    cy.get(`#${SUCCESS_CONTENT_ID}`).should('be.visible');
+
+    cy.get(`#${BACK_BUTTON_ID}`).click();
+
+    cy.get(`#${SUCCESS_CONTENT_ID}`).should('not.exist');
+
+    cy.url().should('include', SIGN_IN_PATH);
+  });
+
+  it('Sign In - Resend email', () => {
+    const { GRAASP, GRAASP_OTHER } = MEMBERS;
+    cy.visit(SIGN_IN_PATH);
+
+    cy.intercept(API_ROUTES.SIGN_IN_ROUTE, ({ reply }) => {
+      return reply({
+        statusCode: StatusCodes.NO_CONTENT,
+      });
+    });
+
+    // Signing in with a valid email
+    cy.signInAndCheck(GRAASP_OTHER);
+    cy.get(`#${BACK_BUTTON_ID}`).click();
+
+    cy.signInAndCheck(GRAASP);
+
+    // checks so request body contains correct email
+    cy.intercept(API_ROUTES.SIGN_IN_ROUTE, ({ body }) => {
+      expect(body.email).to.eq(GRAASP.email);
+    });
+
+    cy.get(`#${RESEND_EMAIL_BUTTON_ID}`).click();
```

Comment:
You mentioned that you disabled the button once clicked, I think it's a really good idea. We can create a new issue with the idea of the timer to deal with later. 
So involving that I would check that the button is indeed disabled once clicked.